test/prism_regression/def_block_param.rb

```diff
@@ -1,9 +1,5 @@
 # typed: false
 
-def foo(&blk); end
+def foo(&my_block_param); end
 
 def foo(&); end
-
-def foo(&blk)
-  super(&blk)
```

Comment:
Calling with a block arg is tested separately in `call_block_param.rb` (which I realize now should be `_arg.rb`